### ISSUE TITLE
Security Update : Bump Validator Version to 13.6.0

### ISSUE THREAD

**Issue by sanknmFinicity** [leepowelldev/mongoose-validator#53]
PRISMA Cloud Reported Vulnerability : 

PRISMA-2021-0063

**Comment by Prepaid2Coin-Cory** [comment]
Just need a merge on https://github.com/leepowelldev/mongoose-validator/pull/52

**Comment by Prepaid2Coin-Cory** [comment]
@leepowelldev Could you merge the above Pull Request? If not could you give a maintainer write permissions such as myself.

**Comment by leepowelldev** [comment]
Happy to merge, but I wonder if this jump should bump this package to a new major version - as I’m unsure if there’s any breaking changes between 10 and 13. An alternative is to move validator package to a peer dependency?

**Comment by Prepaid2Coin-Cory** [comment]
@leepowelldev Yes it should be a major version. I updated the Pull Request to make the change from dependencies to peerDependencies.

**Comment by leepowelldev** [comment]
Updates in master - will aim to release this evening

**Comment by Prepaid2Coin-Cory** [comment]
@leepowelldev Thank you good sir!

**Comment by Prepaid2Coin-Cory** [comment]
Feel free to close this whenever
